spec/convertion_by_format/check_open_xlsx_by_screen_spec.rb

```diff
@@ -24,13 +27,13 @@
       response = converter.perform_convert(url: file_uri(file_path), outputtype: 'png')
       expect(response[:url].nil?).to be_falsey
       expect(response[:url].empty?).to be_falsey
-      image_size = ImageHelper.get_image_size(response[:url])
-      expect(image_size).to be > StaticData::MIN_XLSX_IMAGE_SIZE
+      @image_size = ImageHelper.get_image_size(response[:url])
+      expect(@image_size).to be > StaticData::MIN_XLSX_IMAGE_SIZE
     end
   end
 
   after :each do |example|
     FileHelper.clear_dir('files_tmp')
-    palladium.add_result_and_log(example, image_size)
+      palladium.add_result_and_log(example, @image_size)
```

Comment:
Extra space in start of line

Reply:
Shouldn't rubocop fix that? Is rubocop initalized and overcommit installed?